src/server.h

```diff
@@ -1069,10 +1074,14 @@ struct redisObject {
     unsigned iskvobj : 1;   /* 1 if this struct serves as a kvobj base */
     unsigned expirable : 1; /* 1 if this key has expiration time attached.
                              * If set, then this object is of type kvobj */
-    unsigned refcount : OBJ_REFCOUNT_BITS;
+    unsigned extra : 14;
+    redisAtomic unsigned short refcount;
```

Comment:
do we have performance regression in single thread mode?

Reply:
there is no different locally, but we can wait for @filipecosta90 's fully report.